CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ project(unified-runtime VERSION 0.5.0)
 include(GNUInstallDirs)
 include(CMakePackageConfigHelpers)
 include(CTest)
+include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/helpers.cmake)
```

Comment:
Might be a good idea to add this path to `CMAKE_MODULES_PATH`

Reply:
So something like this @veselypeta?

```cmake
list(APPEND CMAKE_MODULES_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
include(helpers)
```

Reply:
Yes this was my idea. It might be useful if we add more modules in this directory.

Reply:
Done.